cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/VersionPropertiesFileUtils.java

```diff
@@ -0,0 +1,46 @@
+package cloud.foundry.cli.crosscutting.mapping;
+
+import cloud.foundry.cli.crosscutting.logging.Log;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Utils class for reading information from property files.
+ */
+public class VersionPropertiesFileUtils {
+
+    private static final Log log = Log.getLog(VersionPropertiesFileUtils.class);
+
+    /**
+     * Name of the property file, which contains the version of our application
+     */
+    private static final String VERSION_PROPERTIES = "version.properties";
+
+    /**
+     * The key of the version in the properties file
+     */
+    private static final String VERSION = "version";
+
+
+    /**
+     * Determines the API-Version from our application
+     * If it fails, the API Version will be marked as NOT_FOUND and the error will be logged.
+     *
+     * @return API-Version
+     */
+    public static String determineApiVersion(ResourceProvider resourceProvider, Properties properties) {
+        InputStream inputStream = resourceProvider.getInputStreamFromResourceFile(VERSION_PROPERTIES,
+                VersionPropertiesFileUtils.class);
+        try {
+            properties.load(inputStream);
+            String value = properties.getProperty(VERSION);
+            return value;
```

Comment:
instead save the value in value variable, you can return it directly
```suggestion
            return  properties.getProperty(VERSION);
```